js/components/question-for-class.js

```diff
@@ -32,22 +34,30 @@ export default class QuestionForClass extends Component {
     onSelectChange(question.get('key'), event.target.checked)
   }
 
+  handleStickyStateChange(isSticky) {
+    this.setState({isSticky: isSticky})
+  }
+
   render() {
     const { question } = this.props
-    const { answersVisible } = this.state
+    const { answersVisible, isSticky } = this.state
     return (
-      <div className={`question ${question.get('visible') ? '' : 'hidden'}`}>
-        <div className='question-header'>
-          <input type='checkbox' checked={question.get('selected')} onChange={this.handleCheckboxChange}/>
-          Question #{question.get('questionNumber')}
-          <a className='answers-toggle' onClick={this.toggleAnswersVisibility}>
-            {answersVisible ? 'Hide responses' : 'Show responses'}
-          </a>
+      <StickyContainer>
+        <div className={`question ${question.get('visible') ? '' : 'hidden'}`}>
+          <Sticky className="sticky-question-header" onStickyStateChange={this.handleStickyStateChange.bind(this)}>
+            <div className={`question-header ${isSticky ? 'stuck-question-header' : ''}`}>
```

Comment:
`<Sticky>` component automatically receives `.sticky` CSS class when it's stuck. So, `.stuck-question-header` could be probably replaced with `.sticky .question-header` in CSS and no state handing would is necessary. Not a big deal, but I'm assuming that eventually we'll also add sticky page and activity headers, so it could simplify a few components.